tracer/src/Datadog.Trace/IAST/SensitiveData/EvidenceRedactor.cs

```diff
@@ -0,0 +1,107 @@
+// <copyright file="EvidenceRedactor.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+#if !NETCOREAPP3_1_OR_GREATER
+using System.Text.RegularExpressions;
+#endif
+using Datadog.Trace.Logging;
+#if NETCOREAPP3_1_OR_GREATER
+using Datadog.Trace.Vendors.IndieSystem.Text.RegularExpressions;
+#endif
+
+#nullable enable
+
+namespace Datadog.Trace.Iast.SensitiveData;
+
+internal class EvidenceRedactor
+{
+    private readonly Regex _keysRegex;
+    private readonly Regex _valuesRegex;
+    private readonly TimeSpan _timeout;
+    private readonly IDatadogLogger? _logger;
+
+    private Dictionary<string, ITokenizer> _tokenizers = new Dictionary<string, ITokenizer>
+    {
+        { VulnerabilityTypeName.SqlInjection, new SqlInjectionTokenizer() },
+        { VulnerabilityTypeName.LdapInjection, new LdapTokenizer() },
+        { VulnerabilityTypeName.CommandInjection, new CommandTokenizer() },
```

Comment:
Should we need a tokenizer for path traversal?

Reply:
Yes, indeed. It's in the works.